parts/kubernetesmastercustomdata.yml

```diff
@@ -7,8 +7,29 @@ packages:
  - jq
  - traceroute
 
+disk_setup:
+  /dev/sdc:
+      table_type: gpt
+      layout: true
+      overwrite: true
+
+fs_setup:
+- device: /dev/sdc1
+  filesystem: ext4
+  overwrite: true
+  
+mounts:
+- - /dev/sdc1
+  - /var/lib/etcddisk
+      
 runcmd:
-- /bin/echo DAEMON_ARGS=--name "<<<variables('masterVMNames')[copyIndex()]>>>" --initial-advertise-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --listen-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --advertise-client-urls "<<<variables('masterEtcdClientURLs')[copyIndex()]>>>" --listen-client-urls "<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>" --initial-cluster-token "k8s-etcd-cluster" --initial-cluster "<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>>" --initial-cluster-state "new" | tee -a /etc/default/etcd
+- /bin/echo DAEMON_ARGS=--name "<<<variables('masterVMNames')[copyIndex()]>>>" --initial-advertise-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --listen-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --advertise-client-urls "<<<variables('masterEtcdClientURLs')[copyIndex()]>>>" --listen-client-urls "<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>" --initial-cluster-token "k8s-etcd-cluster" --initial-cluster "<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>> --data-dir "/var/lib/etcddisk"" --initial-cluster-state "new" | tee -a /etc/default/etcd
+- sudo /bin/chmod -R 0777  /var/lib/etcddisk
+- systemctl stop etcd
+- sudo rm -rf /var/lib/etcd/default
```

Comment:
Oh, I didn't realize etcd was activating itself by default after install, so weird that Ubuntu does that.

I'm fine with it either way, just wanted to understand. If this mirrors current behavior and you're iterating on it next, it seems fine to leave it as is.